test/Cases.hs

```diff
@@ -41,6 +41,7 @@ spec =
     , runTestForFile "048rgb2.solc" specFolder
     , runTestForFile "06comp.solc" specFolder
     , runTestForFile "09not.solc" specFolder
+    , runTestForFile "092revert.solc" specFolder
```

Comment:
we should also add the longString test here.

Reply:
done